annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -114,6 +120,15 @@ public AnnotationController(VariantAnnotationService variantAnnotationService,
             postEnrichmentService.registerEnricher(isoformOverrideSource, enricher);
         }
 
+        // ignore any other value than "full" or "summary"
+        if (cancerHotspots != null &&
+            (cancerHotspots.equalsIgnoreCase("full") || cancerHotspots.equalsIgnoreCase("summary")))
```

Comment:
A more general comment about the code, I notice a lot of hardcoded strings. Maybe some of these values could be final strings in some class or an enum? Probably shouldn't be addressed in this PR